### DISCUSSION TITLE
Fields like name, displayName in package.json

### DISCUSSION THREAD

**Discussion by GitToTheHub** [apache/cordova#375] [Q&A]
Hi, when I start a new Project from the HelloWorld-Template, I saw a lot of fields in `package.json`, which I'm not sure, if i have to maintain them:
```
  "name": "com.example.hello",
  "displayName": "HelloWorld",
  "version": "1.0.0",
  "description": "A sample Apache Cordova application that responds to the deviceready event.",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [
    "ecosystem:cordova"
  ],
  "author": "Apache Cordova Team",
  "license": "Apache-2.0",
```

When I edit by hand some of the corresponding fields in `config.xml` like
```
    <name>MyApp</name>
    <description>This is my description</description>
```
they aren't reflected to `package.json`, when I do a `cordova prepare`. Do I have to care about these fields? Can I remove them, or should I let them there, as they are, or have I to configure these fields via the CLI?

**Comment by breautek** [comment]
In a cordova app, many "descriptive" fields including `name`, `description` aren't really used. They are used by NPM and I think might be required by NPM to be defined at the very least but isn't used by Cordova, at this not at this time.

With the exception of plugin/dependency management, Cordova still uses the `config.xml` for all configurations related to the app. Platform/plugin details and plugin variables (defined when installing the plugin via `--VARIABLE` arguments) have been moved to package.json under a `cordova: {...}` object.

**Reply by GitToTheHub** [reply]
Tanks for your answer. So if I change the app name, do i have to change it also in package.json?
